src/main/java/sirbu/ZipCodeTest.java

```diff
@@ -25,25 +29,22 @@ public void setUp() {
     }
 
 
-    @After
-    @SneakyThrows
-    public void tearDown() {
-        zipCodeClient.close();
-    }
-
     @Test
     @SneakyThrows
     @Issue("Zip Code")
     @Step("Get all available zip codes")
     public void testGetZipCodes() {
-        CloseableHttpResponse response = zipCodeClient.getZipCodesResponse();
-        int statusCode = response.getStatusLine().getStatusCode();
+        Response response = given()
+                .header("Authorization", "Bearer " + AuthProvider.getInstance().getReadToken())
+                .get(zipCodeClient.getZipCodesUrl());
```

Comment:
This part should be moved to ZipCodeClient getZipCodeResponse method. Test shouldn't know anything about tokens.